README.md

```diff
@@ -1,13 +1,13 @@
-# Thesis Content Editing System
+# Thesis
 
 [![Build Status](https://semaphoreci.com/api/v1/projects/0540dbb0-887a-45dd-9190-baa19b2ca9fb/816876/badge.svg)](https://semaphoreci.com/ir/thesis-phoenix)
 
-Thesis is a lightweight and flexible Elixir/Phoenix hex package for quickly and easily
-adding content editing to any page on a Phoenix website.
+Thesis is a lightweight and flexible Elixir/Phoenix CMS for quickly and easily
+adding content editing to any page on a Phoenix website, as well as creating new
+dynamically routed pages. It's ideal for either adding limited editing support to
+existing Phoenix websites or building dynamic websites.
```

Comment:
I've decided to go ahead and call Thesis a CMS. The functionality may be lighter than other CMS's, but it can function as one in the majority of cases.